scarb/src/ops/package.rs

```diff
@@ -285,6 +291,39 @@ fn prepare_archive_recipe(pkg: &Package, opts: &PackageOpts) -> Result<ArchiveRe
     Ok(recipe)
 }
 
+fn run_verify(ws: &Workspace<'_>, pkg: &Package, tar: &FileLockGuard) -> Result<()> {
```

Comment:
```suggestion
fn run_verify(pkg: &Package, tar: &FileLockGuard, ws: &Workspace<'_>) -> Result<()> {
```

we used to always put `ws` or `config` as the last argument